### ISSUE TITLE
<ix-select> Show selected value after leave the "filter" field

### ISSUE THREAD

**Issue by csanadEvo** [siemens/ix#1430]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

- I have a selected value in the ix-select field
![image](https://github.com/user-attachments/assets/536eef54-5f08-4659-9162-49c7cc287ec9)
- Click in the field and start typing (filter the list)
![image](https://github.com/user-attachments/assets/60046cbe-d0bc-4361-89f9-804c8469d181)
- Leave the field without select any new value. The search text will be shown instead of the selected value
![image](https://github.com/user-attachments/assets/d99afd62-bde9-4066-a4ac-31eca0be1842)
- The field should show the selected value.

--
Also would be nice to have a property for disable the filter/search functionality (users can't click and type into the field).

Thanks,
  Csanad

### What type of frontend framework are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.1.0

### Code to produce this issue.

```shell
<ix-select
            hideListHeader="true"
            i18nPlaceholder=""
            formControlName="formControlName"
            (ngModelChange)="updateValue($event)"
        >
        <ix-select-item
              *ngFor="let item of availableOptions"
              [value]="item.key"
              [label]="item.value"
            ></ix-select-item>
        </ix-select>
```

**Comment by matthiashader** [comment]
Hello @csanadEvo - did you try to update your iX Version?
I have  tried to recreate the behavior with the newest version (2.4.1), where it doesnt seem to appear:
https://stackblitz.com/edit/9tarsy?file=src%2Fapp%2Fselect.ts

**Comment by matthiashader** [comment]
Fixed in latest version (2.5.0).
